spec/features/password_reset_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'the token password reset process', type: :feature do
+  it 'edits the password using reset token' do
```

Comment:
tests look good but could we could format the code better. Adding spaces between groups of sections.